src/complex/Filter/IParameter.hpp

```diff
@@ -18,18 +18,23 @@
 
 namespace complex
 {
+namespace Constants
+{
 /**
  * @brief These constants can be used as standard error codes when parsing JSON
  */
-namespace FilterParameter::Constants
-{
-inline constexpr int32 k_Json_Missing_Entry = -100;
-inline constexpr int32 k_Json_Value_Not_Object = -101;
-inline constexpr int32 k_Json_Value_Not_String = -102;
-inline constexpr int32 k_Json_Value_Not_Integer = -103;
-inline constexpr int32 k_Json_Value_Not_Unsigned = -104;
-inline constexpr int32 k_Json_Value_Not_Enumeration = -105;
-} // namespace FilterParameter::Constants
+inline constexpr int32_t k_Json_Missing_Entry = -100;
+inline constexpr int32_t k_Json_Value_Not_Object = -101;
+inline constexpr int32_t k_Json_Value_Not_String = -102;
+inline constexpr int32_t k_Json_Value_Not_Integer = -103;
+inline constexpr int32_t k_Json_Value_Not_Unsigned = -104;
+inline constexpr int32_t k_Json_Value_Not_Enumeration = -105;
+inline constexpr int32_t k_Json_Value_Not_Value_Type = -106;
+
+inline constexpr int32_t k_Validate_Empty_Value = -201;
+inline constexpr int32_t k_Validate_DuplicateValue = -202;
+inline constexpr int32_t k_Validate_ExistingValue = -203;
```

Comment:
```suggestion
inline constexpr int32 k_Json_Missing_Entry = -100;
inline constexpr int32 k_Json_Value_Not_Object = -101;
inline constexpr int32 k_Json_Value_Not_String = -102;
inline constexpr int32 k_Json_Value_Not_Integer = -103;
inline constexpr int32 k_Json_Value_Not_Unsigned = -104;
inline constexpr int32 k_Json_Value_Not_Enumeration = -105;
inline constexpr int32 k_Json_Value_Not_Value_Type = -106;

inline constexpr int32 k_Validate_Empty_Value = -201;
inline constexpr int32 k_Validate_DuplicateValue = -202;
inline constexpr int32 k_Validate_ExistingValue = -203;
```

These changes need to be matched with the ones from master.